recipes/bison/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import glob
+import os
+import shutil
+
+
+class ConanFileDefault(ConanFile):
+    name = "bison"
+    url = "https://github.com/conan-io/conan-center-index   "
+    homepage = "https://www.gnu.org/software/bison/"
+    description = "Bison is a general-purpose parser generator"
+    topics = ("conan", "bison", "parser")
+    license = "GPL-3.0-or-later"
+    exports_sources = ["patches/*.patch"]
+    _source_subfolder = "source_subfolder"
+    requires = ("m4/1.4.18", "flex/2.6.4")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "bison-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        self._build_configure()
+
+    def _build_configure(self):
+        true = tools.which("true") or "/bin/true"
+        args = ["HELP2MAN=%s" % true, "MAKEINFO=%s" % true, "--disable-nls"]
+        build = None
+        host = None
+
+        env_build = AutoToolsBuildEnvironment(self)
+        with tools.chdir(self._source_subfolder):
+            tools.replace_in_file("Makefile.in",
+                                  "dist_man_MANS = $(top_srcdir)/doc/bison.1",
+                                  "dist_man_MANS =")
+            tools.replace_in_file(os.path.join("src", "yacc.in"),
+                                  "@prefix@",
+                                  "${}_ROOT".format(self.name.upper()))
+            tools.replace_in_file(os.path.join("src", "yacc.in"),
+                                  "@bindir@",
+                                  "${}_ROOT/bin".format(self.name.upper()))
+
+            env_build.configure(args=args, build=build, host=host)
+            env_build.make()
+            env_build.install()
```

Comment:
Can we move `install()` to `package()` or is this problematic here?

Reply:
And maybe `make()` to `build()` itself to be closer to how we handle CMake builds

Reply:
@Croydon 
`AutoToolsBuildEnvironment` does not have a `build()` method. `make()` will have to do 😄

Reply:
I meant moving the `env_build.make()` in `build()` instead of leaving it in `_build_configure`